native/Makefile

```diff
@@ -246,8 +246,21 @@ LIBS=psapi.lib
 ARSFX=.lib
 ifeq ($(ARCH),amd64)
 CC+= -m64
-endif
-FFI_CONFIG+= && rm -f include/ffitarget.h && cp $(FFI_SRC)/include/*.h $(FFI_SRC)/src/x86/ffitarget.h include
+FFI_TARGET=$(FFI_SRC)/src/x86/ffitarget.h
+else ifeq ($(ARCH),aarch64)
+# Force $(CC) fallback; avoid arch mismatch
+WINDRES=false
+# Disable mingw; no aarch64 support
+MINGW=
+FFI_CONFIG+=--host=aarch64-cygwin
+# Skip building assembly callback functions
+CDEFINES+= -DASMFN_OFF
+DLLCB=
```

Comment:
So the new fix is a bit more elegant.  If we wipe out `DLLCB` var, the `EXTRAOBJS+=` lower in the Makefile won't add it, which is how non-`amd64` does it anyway.  The custom precompiler flag is still needed.  We could detect an aarch64 compiler, but it's more controlled this way and allows a farily quick toggle-off when mingw adds aarch64 support.